.github/workflows/release.yml

```diff
@@ -4,7 +4,9 @@ on:
   push:
     tags:
       - v[0-9]+.[0-9]+.[0-9]+
-
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
This probably shouldn't have this in place, as all release jobs must be completed entirely

Reply:
86b245eae7aab630099765a786894628cf3a8a09